man/docker-network-create.1.md

```diff
@@ -0,0 +1,137 @@
+* A cluster of hosts with connectivity to the key-value store.
+* A properly configured Engine `daemon` on each host in the cluster.
+
+The `docker daemon` options that support the `overlay` network are:
+
+* `--cluster-store`
+* `--cluster-store-opt`
+* `--cluster-advertise`
+
+It is also a good idea, though not required, that you install Docker Swarm on
+to manage the cluster. Swarm provides sophisticated discovery and server
+management that can assist your implementation. Once you have prepared the
+`overlay` network prerequisites you simply choose a Docker host in the cluster
+and issue the following to create the network:
+```
+$ docker network create -d overlay my-multihost-network
+```
+
+Network names must be unique. The Docker daemon attempts to identify naming
+conflicts but this is not guaranteed. It is the user's responsibility to avoid
+name conflicts.
+
+## Connect containers
+
+When you start a container use the `--net` flag to connect it to a network.
+This adds the `busybox` container to the `mynet` network.
+
+```
+$ docker run -itd --net=my-multihost-network busybox
+```
+
+If you want to add a container to a network after the container is already
+running use the `docker network connect` subcommand.
+
+You can connect multiple containers to the same network. Once connected, the
+containers can communicate using only another container's IP address or name.
+For `overlay` networks or custom plugins that support multi-host connectivity,
+containers connected to the same multi-host network but launched from different
+Engines can also communicate in this way.
+
+You can disconnect a container from a network using the `docker network
+disconnect` command.
+
+## Specifying advanced options
+
+When you create a network, Engine creates a non-overlapping subnetwork for the network by default. You can override this default and specify a subnetwork directly using the the `--subnet` option. On a `bridge` network you can only create a single subnet:
+
+```
+docker network create -d --subnet=192.168.0.0/16
+```
+Additionally, you also specify the `--gateway` `--ip-range` and `--aux-address` options.
+
+```
+network create --driver=bridge --subnet=172.28.0.0/16 --ip-range=172.28.5.0/24 --gateway=172.28.5.254 br0
+```
+
+If you omit the `--gateway` flag the Engine selects one for you from inside a
+preferred pool. For `overlay` networks and for network driver plugins that
+support it you can create multiple subnetworks.
+
+```
+docker network create -d overlay
+  --subnet=192.168.0.0/16 --subnet=192.170.0.0/16
+  --gateway=192.168.0.100 --gateway=192.170.0.100
+  --ip-range=192.168.1.0/24
+  --aux-address a=192.168.1.5 --aux-address b=192.168.1.6
+  --aux-address a=192.170.1.5 --aux-address b=192.170.1.6
+  my-multihost-newtork
+```
+Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
+
+# OPTIONS
+**--aux-address=map[] **     
```

Comment:
One space too much at the end ? (not sure if it's a big deal or not for man pages, but looks strange on github :stuck_out_tongue:)